src/js/components/sharedComponents/filterSidebar/FilterExpandButton.jsx

```diff
@@ -125,14 +93,21 @@ export default class FilterExpandButton extends React.Component {
                     aria-expanded={this.props.arrowState === 'expanded'}
                     aria-describedby={ariaDescription}>
                     {icon}
-                    <div className="filter-toggle__name">
-                        {this.props.name}
+                    <div className={`filter-toggle__name ${this.props.className}`}>
```

Comment:
Wont this add an extra space if there is no className or even add undefined to the className? Can we change this to an if statement or method?

Reply:
If anything it will just an empty space.

Reply:
Yeah, I just think it throws off our scrapers we use for the testing team.

Reply:
It could also say `filter-toggle__name undefined`

Reply:
Got it.